she_codes_news/news/views.py

```diff
@@ -31,3 +43,9 @@ class AddStoryView(generic.CreateView):
     def form_valid(self, form):
         form.instance.author = self.request.user
         return super().form_valid(form)
+
+    def get_initial(self):
+        initial = super().get_initial()
+        tz_name = get_tz_name(self.request)
+        initial['pub_date'] = datetime.datetime.now(ZoneInfo(tz_name)).strftime("%Y-%m-%dT%H:%M")
```

Comment:
or actually, even simpler - 

```
initial['pub_date'] = timezone.now().strftime("%Y-%m-%dT%H:%M") 
```

Reply:
this was legacy, so yes good catch

Reply:
i removed this, and it didn't actually work :/